keps/sig-node/2043-pod-resource-concrete-assigments/README.md

```diff
@@ -339,7 +341,7 @@ Feature only collects data when requests comes in, data is then garbage collecte
 ### Troubleshooting
 
 * **How does this feature react if the API server and/or etcd is unavailable?**: No effect.
-* **What are other known failure modes?** No known failure modes
+* **What are other known failure modes?** feature gate disabled: the API will always return a well-known error. In normal operation, the API is expected to never return error and always return a valid response, because it utilizes internal kubelet data which is always available. Bugs may lead to the API to return unexpected errors, or to return inconsistent data. Consumers of the API should treat unexpected errors as bugs of this API.
```

Comment:
Presumably feature gate is going away if we go to stable?

Reply:
(or rather locking to enabled for now)

Reply:
Yes. Since not every worker node requires this feature (I mean whole podresources interface), I think, feature gate could be used to enable/disable interface.